docs/tutorial.md

```diff
@@ -1,3 +1,55 @@
 # Tutorial
 
-TODO Use demo_plugin to describe how to do stuff.
+To help illustrate the concepts of NewHELM, we provide a series of functional (if silly) examples in the `demo_plugin` folder. This tutorial walks through those examples.
+
+# Tests
+
+## Creating a basic Test
+
+[Demo: DemoSimpleQATest](../demo_plugin/newhelm/tests/demo_01_simple_qa_test.py)
+
+Let's say we want to create a Test where we send a bunch of questions to the SUT, and expect it to respond with specific answers. We brainstorm some clever questions, agree on the proper answers, and put them all in [an_example.jsonl](https://storage.googleapis.com/mlc-aisafety-scratch-dev-7292542/an_example.jsonl).
+
+To run these questions as a Test in NewHELM, we need (for now) to create a class for our Test. Let's call our Test `DemoSimpleQATest`.
+Since this fits as a [Prompt Response Tests](prompt_response_tests.md) we can have it inherit from `BasePromptResponseTest`. We now have several abstract methods we need to define.
```

Comment:
Small nit: "a Prompt Response Tests" is awkward; perhaps "Test" is better.

Reply:
I'm trying to call out that this is a specific kind of Test, albeit the only kind of Test we currently support. So just "Test" here doesn't convey the important information.

Is the problem the link highlighting?

Reply:
Sorry, I should have phrased that differently. It's just the agreement between "a" and "Tests". So I'm suggesting "a Prompt Response Test".

Reply:
Ah, yes, Fixed.